Makefile.js

```diff
@@ -279,17 +279,16 @@ target.gensite = function() {
 
     echo("Generating eslint.org");
 
-    if (test("-d", DOCS_DIR)) {
-        rm("-r", DOCS_DIR);
```

Comment:
Good point. I don't know... I guess it's not really all that critical to remove those old files, as long as links are updated and manual clean-up can be done once in a while. But would be nice if it would be automated... The only way I can think about doing it, is separating static and dynamic content into separate directories. But that's not great for SEO.
I can just wipe out "rules" directory. That's the only one that should be changing at any reasonable rate.